mhkit/wave/resource.py

```diff
@@ -824,32 +944,27 @@ def wave_length(k):
 
     Parameters
     -------------
-    k: pandas Dataframe
+    k: int, float, numpy ndarray, pandas Series, pandas DataFrame, xarray DataArray, or xarray Dataset
         Wave number [1/m] indexed by frequency
 
     Returns
     ---------
-    l: float or array
+    l: input type
```

Comment:
To be explicit I think this would be input type of k.  Took me a second to realize what this meant and then that there was only 1 input.  Maybe we copy pasta the k input types and then state in the description how it is decided.

Reply:
Good point. I copied the output type possibilities and specified that it is determined by the type of k